### PR TITLE
fix: added temp Navbar

### PR DESCRIPTION
**Pull request by luanvdw**
The [PR](https://github.com/prisma/docs/pull/2297) to make use of Lens components, that were built with Tailwind, requires a larger discussion and planning for a successful roll out.

This PR serves as a temporary solution to have the new <Navbar /> component within the Docs project.

### CODE REVIEW

File: src/components/navbar/MenuList.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        svg {
+          margin-right: 0.5rem;
+          path {
+            fill: ${theme.colors.gray800};
+          }
+        }
+      }
+      &:focus {
+        outline: none;
+      }
```
Comment (molebox) [inline src/components/navbar/MenuList.tsx]: Do we want to add some additional styling if we are removing the outline on focus? Keyboard users wont see any difference now

File: src/components/navbar/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+const MobileMenuBtn = styled.button`
+  background-color: transparent;
+  display: flex;
+  padding: 0;
+  margin: 0;
+  border: 0;
+  &:focus,
+  &:active {
+    outline: none;
+  }
```
Comment (molebox) [inline src/components/navbar/Navbar.tsx]: Do we want to add some additional styling if we are removing the outline on focus? Keyboard users wont see any difference now

**Review by molebox — COMMENTED** [review]
Looks good! Just left a couple of comments

**Review by nilubava — APPROVED** [review]
since this is temporary, looks good to me :)

File: src/components/navbar/MenuList.tsx (hunk 1 above)
Comment (luanvdw) [reply src/components/navbar/MenuList.tsx]: 100%, re-added the default focus ring for this solution.

File: src/components/navbar/Navbar.tsx (hunk 2 above)
Comment (luanvdw) [reply src/components/navbar/Navbar.tsx]: 100%, re-added the default focus ring for this solution.

**Pull request merged**